tools/openshift-ci/Dockerfile

```diff
@@ -5,7 +5,13 @@ LABEL maintainer="OCP QE Team"
 USER root
 
 ARG YQ_VERSION="v4.30.8"
-RUN set -x && \
+
+#RUN cd /etc/yum.repos.d/
+#RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+#RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```

Comment:
Shall we remove them if they are not needed ?

Reply:
ack, all comments that are not relevant should be removed here.